graphql_compiler/typedefs.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020-present Kensho Technologies, LLC.
+from typing import Union
+
+from graphql import GraphQLList, GraphQLNonNull, GraphQLScalarType
+
+
+# The compiler's supported GraphQL types for query arguments. The GraphQL type of a query argument
+# is the type of the field that the argument references. Not to be confused with the GraphQLArgument
+# class in the GraphQL core library.
+GraphQLQueryArgumentType = Union[
```

Comment:
I think that it would make sense to move `QueryStringWithParameters` and `ASTWithParameters` from global_utils.py to here in a subsequent diff. Thoughts?